echopype/tests/mask/test_mask.py

```diff
@@ -2006,3 +2010,180 @@ def test_echoview_mincan_no_linking():
     # Label the mask and confirm they are separate components (not connected)
     _, nlab = ndi.label(mask.values, structure=np.ones((3, 3), dtype=bool))
     assert nlab == 2
+
+# test for single target detections
+
+REQ = {
+    "channel": "chan1",
+    "pulse_length": 1e-3,             # 1 ms dummy
+    "timeSampleInterval_usec": 40.0,  # dummy
```

Comment:
Not sure where this would be used, but maybe also change this to using snake_case?